internal/pkg/skuba/kubeadm/configmap_test.go

```diff
@@ -209,6 +209,7 @@ func TestGetAPIEndpointsFromConfigMap(t *testing.T) {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			gotAPIEndpoints, err := GetAPIEndpointsFromConfigMap(tt.client)
+			sort.Strings(gotAPIEndpoints)
```

Comment:
Thanks, I forgot to add to this test case.
I only add to https://github.com/SUSE/skuba/blob/c965dff9bd2f1af2234daf8a9d8df3cdc84f466e/internal/pkg/skuba/kubeadm/configmap_test.go#L317-L322